packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -200,38 +203,163 @@ mixin RendererBinding on BindingBase, ServicesBinding, SchedulerBinding, Gesture
+    }
+  );
+
+  /// Deprecated. Will be removed in a future version of Flutter.
+  ///
+  /// This is typically the root of the render tree bootstrapped by [runApp].
+  ///
+  /// However, by default this render view is not associated with any
+  /// [PipelineOwner] and therefore isn't considered during frame production.
+  /// It is also not registered with this binding via [addRenderView].
+  /// Wiring this up is left as a responsibility for a higher level. The
+  /// [WidgetsBinding], for example, sets this up in
+  /// [WidgetsBinding.wrapWithDefaultView], which is called indirectly from
+  /// [runApp].
+  ///
+  /// Apps, that don't use the [WidgetsBinding] or don't call [runApp] (or
+  /// [WidgetsBinding.wrapWithDefaultView]) must manually assign a
+  /// [PipelineOwner] to this [RenderView], make sure the pipeline owner is part
+  /// of the pipeline owner tree rooted at [rootPipelineOwner], and call
+  /// [addRenderView] if they want to use this deprecated property.
+  ///
+  /// Instead of interacting with this deprecated property, consider using
+  /// [renderViews] instead, which contains all [RenderView]s managed by the
+  /// binding.
+  @Deprecated(
+    'Consider using RendererBinding.renderViews instead as the binding may manage multiple RenderViews. '
+    'This feature was deprecated after v3.10.0-12.0.pre.'
+  )
+  // TODO(goderbauer): When this deprecated property is removed also delete the _ReusableRenderView class.
+  late final RenderView renderView = _ReusableRenderView(
+    view: platformDispatcher.implicitView!,
+  );
+
+  /// Creates the [PipelineOwner] that serves as the root of the pipeline owner
+  /// tree ([rootPipelineOwner]).
+  ///
+  /// {@template flutter.rendering.createRootPipelineOwner}
+  /// By default, the root pipeline owner is not setup to manage its own
+  /// [RenderView] and in this configuration nothing must be assigned to
+  /// [PipelineOwner.rootNode] of [rootPipelineOwner]. If necessary,
+  /// [createRootPipelineOwner] may be overridden to create a root pipeline
+  /// owner configured to manage its own [RenderView].
+  ///
+  /// In typical use, children are added to the root pipeline owner (via
+  /// [PipelineOwner.adoptChild]). Those children typically do each manage their
+  /// own [RenderView] and produce distinct render trees which render their
+  /// content into the [FlutterView] associated with that [RenderView].
+  /// {@endtemplate}
+  PipelineOwner createRootPipelineOwner() {
+    return PipelineOwner(onSemanticsUpdate: (ui.SemanticsUpdate update) {
+      assert(() {
+        throw FlutterError.fromParts(<DiagnosticsNode>[
+          ErrorSummary(
+            'The global pipeline owner produced an unexpected semantics update.',
+          ),
+          ErrorDescription(
+            'By default, the RendererBinding.rootPipelineOwner is not configured '
+            'to handle semantics because it is not expected to own a root node.',
```

Comment:
In my first read through, I thought "root node" was referring to a "semantics node" instead of `PipelineOwner.rootNode`. Perhaps consider saying "root render object" instead?

Reply:
As per your next suggestion, I removed this error message altogether.